aegis-web/yo/app/styles/filebrowser.css

```diff
@@ -114,6 +116,7 @@
   padding: 0;
   position: relative;
   box-shadow: 1px 2px 7px 1px rgba(0, 0, 0, 0.1);
+  height: 100%;
```

Comment:
This property is breaking the bottom shadow of the files box. I don't know why you added it so i can judge if you need to keep it or not but somehow the shadow should be fixed.
![image](https://user-images.githubusercontent.com/1640100/55372447-13ae5b00-5502-11e9-906c-9b53796d5ca7.png)